### ISSUE TITLE
More than 3k issues in code. 

### ISSUE THREAD

**Issue by Mutaib-yye** [Muhammed-Shiyas/E-Campus#4]
Can't use pubspec.yaml 

Can you tell me step by step procedure (detailed) about how to ensure this works properly in my laptop.

**Comment by Bhargavzz** [comment]
Yes, the dependencies currently used are outdated, resulting in version conflicts. Kindly update the pubspec.yaml to resolve these issues.
